nuitka/nodes/AttributeLookupNodes.py

```diff
@@ -33,9 +33,10 @@ def computeExpression(self, trace_collection):
         )
 
     def mayRaiseException(self, exception_type):
-        return self.subnode_expression.mayRaiseException(
+        subnode_expression = self.subnode_expression
```

Comment:
I am thinking repeated attribute lookups of the same attribute may have a cache in python, do they not? Generally the change looks good to me.